src/qt/locale/bitcoin_nb.ts

```diff
@@ -3935,7 +3935,7 @@ Use this functionality with extreme caution.</source>
     </message>
     <message>
         <source>Tor control port password (default: empty)</source>
-        <translation>Passord for Tor-kontrollport (standardverdi: tom)</translation>
+        <translation>Password for Tor-kontrollport (standardverdi: tom)</translation>
```

Comment:
I understand you're proposing to change "passord" to "password" in the Norwegian translation, but I must advise against this modification. "Passord" is the correct Norwegian term for "password", while "password" is the English word. Norwegian, like many languages, adapts technical terms into its own vocabulary rather than directly using English words. Using "passord" is not only linguistically accurate but also consistent with Norwegian software localization standards. While many Norwegians might understand "password" due to English influence, it's not considered correct in formal Norwegian or official contexts. Maintaining "passord" ensures the translation remains idiomatic and adheres to Norwegian language conventions for technical terminology.